FoundationsMastery/CustomString.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using System;
+using System.Collections.Generic;
+using FoundationsMastery.Interfaces;
+using System.Collections;
+
+namespace FoundationsMastery
+{
+    public class CustomString : ICustomString
+    {
+        public IEnumerable<char> Contents { get; set; }
+
+        public CustomString(char[] mychars)
+        {
+            Contents = mychars;
+        }
+
+        public CustomString()
+        {
+        }
+
+        public int Length
+        {
+            get
+            {
+                return Contents.Count();
+            }
+        }
+
+        public void Clear()
+        {
+            return myString.Clear();
```

Comment:
`myString` is undefined in this context... I think what you want is some way of clearing the `Contents` property on the `this` instance